lighthouse-core/report/html/html-report-assets.js

```diff
@@ -23,6 +23,8 @@ const REPORT_JAVASCRIPT = [
   fs.readFileSync(__dirname + '/renderer/pwa-category-renderer.js', 'utf8'),
   fs.readFileSync(__dirname + '/renderer/report-renderer.js', 'utf8'),
   fs.readFileSync(__dirname + '/renderer/i18n.js', 'utf8'),
+  fs.readFileSync(__dirname + '/renderer/base64.js', 'utf8'),
+  fs.readFileSync(require.resolve('pako/dist/pako_deflate.min.js'), 'utf-8'),
```

Comment:
this is probably problematic for devtools...

Reply:
seems like we could skip deflate in most of our environments and be ok falling back to non-gzipped if compressionstreams aren't supported? now that I say this I wonder how much value all of this gzip business has to begin with...

it's the inflate side where they're sharing it out that it's critical to support if we support deflate anywhere

Reply:
> seems like we could skip deflate in most of our environments and be ok falling back to non-gzipped if compressionstreams aren't supported?

done.

> now that I say this I wonder how much value all of this gzip business has to begin with...

I got 10kb for the treemap options, vs ~60kb if no gzip. Either way, it's a really long URL. And both are too long for every URL shortener service I've tried.

I'm fine without the gzip. @paulirish ?